extension_templates/forecasting.py

```diff
@@ -261,6 +264,31 @@ def _predict_quantiles(self, fh, X=None, alpha=0.5):
         """
         # implement here
 
+    def _predict_var(self, fh, X=None, cov=False):
+        """
+        Compute/return prediction variance for a forecast.
```

Comment:
suggest, in formatting and style, try to be consistent with `_predict`

Reply:
yes it's now `"""Compute/return...`